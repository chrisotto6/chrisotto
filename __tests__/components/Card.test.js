import { render } from '@testing-library/react'
import Card from '@/components/Card'

it('renders card with url unchanged', () => {
  const { container } = render(
    <Card title="test" description="test" imgSrc="/test" href="https://www.google.com" />
  )
  expect(container).toMatchSnapshot()
})

it('renders card without url unchanged', () => {
  const { container } = render(<Card title="test" description="test" imgSrc="/test" />)
  expect(container).toMatchSnapshot()
})
