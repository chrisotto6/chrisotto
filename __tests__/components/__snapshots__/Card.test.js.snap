// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders card with url unchanged 1`] = `
<div>
  <div
    class="md p-4 md:w-1/2"
    style="max-width: 544px;"
  >
    <div
      class="h-full  overflow-hidden rounded-md border-2 border-gray-200 border-opacity-60 dark:border-gray-700"
    >
      <a
        aria-label="Link to test"
        href="https://www.google.com"
        rel="noopener noreferrer"
        target="_blank"
      >
        <span
          style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
        >
          <span
            style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
          >
            <img
              alt=""
              aria-hidden="true"
              src="data:image/svg+xml,%3csvg xmlns=%27http://www.w3.org/2000/svg%27 version=%271.1%27 width=%27544%27 height=%27306%27/%3e"
              style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
            />
          </span>
          <img
            alt="test"
            class="object-cover object-center md:h-36 lg:h-48"
            data-nimg="intrinsic"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
          />
          <noscript />
        </span>
      </a>
      <div
        class="p-6"
      >
        <h2
          class="mb-3 text-2xl font-bold leading-8 tracking-tight"
        >
          <a
            aria-label="Link to test"
            href="https://www.google.com"
            rel="noopener noreferrer"
            target="_blank"
          >
            test
          </a>
        </h2>
        <p
          class="prose mb-3 max-w-none text-gray-500 dark:text-gray-400"
        >
          test
        </p>
        <a
          aria-label="Link to test"
          class="text-base font-medium leading-6 text-primary-500 hover:text-primary-600 dark:hover:text-primary-400"
          href="https://www.google.com"
          rel="noopener noreferrer"
          target="_blank"
        >
          Learn more â†’
        </a>
      </div>
    </div>
  </div>
</div>
`;

exports[`renders card without url unchanged 1`] = `
<div>
  <div
    class="md p-4 md:w-1/2"
    style="max-width: 544px;"
  >
    <div
      class="h-full  overflow-hidden rounded-md border-2 border-gray-200 border-opacity-60 dark:border-gray-700"
    >
      <span
        style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
      >
        <span
          style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
        >
          <img
            alt=""
            aria-hidden="true"
            src="data:image/svg+xml,%3csvg xmlns=%27http://www.w3.org/2000/svg%27 version=%271.1%27 width=%27544%27 height=%27306%27/%3e"
            style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
          />
        </span>
        <img
          alt="test"
          class="object-cover object-center md:h-36 lg:h-48"
          data-nimg="intrinsic"
          decoding="async"
          src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
          style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
        />
        <noscript />
      </span>
      <div
        class="p-6"
      >
        <h2
          class="mb-3 text-2xl font-bold leading-8 tracking-tight"
        >
          test
        </h2>
        <p
          class="prose mb-3 max-w-none text-gray-500 dark:text-gray-400"
        >
          test
        </p>
      </div>
    </div>
  </div>
</div>
`;
